src/ride/models.py

```diff
@@ -19,5 +20,5 @@ class Ride(models.Model):
     is_flexible = models.BooleanField(default=False)
     riders = models.ManyToManyField(Person)
     estimated_cost = models.FloatField(default=None, null=True)
-    path = models.OneToOneField(Path, on_delete=models.CASCADE, primary_key=True)
+    path = models.ForeignKey(Path, on_delete=models.CASCADE)
```

Comment:
one ride may have many paths? i thought a ride would be from point A to point B - what is this for and how does it relate to search?